metadata-ingestion/setup.py

```diff
@@ -77,6 +77,7 @@ def get_long_description():
     "snowflake": sql_common | {"snowflake-sqlalchemy"},
     "oracle": sql_common | {"cx_Oracle"},
     "ldap": {"python-ldap>=2.4"},
+    "looker": {"lkml>=1.1.0", "sql-metadata==1.12.0", "looker-sdk==21.6.0"},
```

Comment:
It might be better for usability to split `looker` and `lookml`, even if they end up being the same deps. I'd like to keep the 1:1 correspondence between source/sink names and pip extra names.

Reply:
Got it! Fixed!